src/sentry/integrations/github/webhook.py

```diff
@@ -92,7 +93,28 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
                 organization_id__in=orgs.keys(),
                 provider=f"integrations:{self.provider}",
                 external_id=str(event["repository"]["id"]),
-            )
+            ).exclude(status=ObjectStatus.HIDDEN)
+
+            if not repos.exists():
+                binding_key = "integration-repository.provider"
```

Comment:
I mentioned this in the other PR, but maybe this process to get the provider should be a library function.